### PR TITLE
Create 10.0.17763.437

### PR DESCRIPTION
**Pull request by torrentu**

### CODE REVIEW

**Comment by torrentu** [comment]
Add version 10.0.17763.437

**Comment by haykoatmaca** [comment]
hi..
create 10.0.18362.267

**Comment by fre4kyC0de** [comment]
https://github.com/fre4kyC0de/rdpwrap/commit/17ca87ad2a35f7cdbe208dad03aced2f9989ca38

**Comment by bober777** [comment]
Pls create 10.0.17763.292

File: 10.0.17763.437 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+bMultimonAllowed.x64 =ECAC8
+ulMaxDebugSessions.x64=ECACC
+bFUSEnabled.x64 =ECAD0
+
+[10.0.17763.437]
+LocalOnlyPatch.x64=1
+LocalOnlyOffset.x64=77A41
+LocalOnlyCode.x64=jmpshort
+SingleUserPatch.x64=1
+SingleUserOffset.x64=1322C
```
Comment (fre4kyC0de) [inline 10.0.17763.437:24]: Incorrect.
Correct would be 1339C

**Comment by fre4kyC0de** [comment]
https://github.com/fre4kyC0de/rdpwrap/commit/275e5475c2f4ce279236ec73864ff03b9689e218

**Pull request closed without merging**